CHANGELOG.md

```diff
@@ -1,16 +1,12 @@
-# nf-core/staramr: Changelog
+# phac-nml/staramrnf: Changelog
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## 0.1.0 - [TBD]
 
-Initial release of nf-core/staramr, created with the [nf-core](https://nf-co.re/) template.
+Initial release of staramrnf, or staramr nextflow pipeline, is a nextflow wrapper of [staramr](https://github.com/phac-nml/staramr/).
 
-### `Added`
+`staramr` (_AMR) scans bacterial genome contigs against the [ResFinder][resfinder-db], [PointFinder][pointfinder-db], and [PlasmidFinder][plasmidfinder-db] databases (used by the [ResFinder webservice][resfinder-web] and other webservices offered by the Center for Genomic Epidemiology) and compiles a summary report of detected antimicrobial resistance genes. The `star|_`in`staramr` indicates that it can handle all of the ResFinder, PointFinder, and PlasmidFinder databases.
```

Comment:
Can you also add the URLs to `ResFinder` and `PointFinder` here like in the README.

Reply:
Yes, I forgot that I just fixed it on README.md 788f0f6